### ISSUE TITLE
CPos.X/Y is undocumented

### ISSUE THREAD

**Issue by Mailaender** [OpenRA/OpenRA#19769]
https://github.com/OpenRA/OpenRA/blob/6e0917169d56ade907b479e870dca4b67c85dcb7/OpenRA.Game/CPos.cs#L134-L145

**Comment by abcdefg30** [comment]
For clarification, I assume this is about those not showing up at https://docs.openra.net/en/latest/release/lua/#cpos ?

**Comment by Mailaender** [comment]
Yes, the property is not visible in any API documentation. I am not sure how to solve this yet.

**Comment by abcdefg30** [comment]
The docs generated for CPos are also just for the global table, i.e. for `CPos.Zero` and `CPos.New`, not for accessing fields on a variable itself. I suppose we'd need a new *properties thing (like actor or player properties). (Coordinate properties? Thing is all other types like `WPos` also don't come up in the docs. -> CPosProperties, WPosProperties?)

**Comment by pchote** [comment]
They are special hardcoded types, so would need special hardcoded documentation.
